modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -20,11 +20,16 @@ endif::[]
 = Creating a manifest object that includes a customized `br-ex` bridge
 
 ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
-As an alternative to using the `configure-ovs.sh` shell script to set a `br-ex` bridge on a bare-metal platform, you can create a `MachineConfig` object that includes an NMState configuration file. The NMState configuration file creates a customized `br-ex` bridge network configuration on each node in your cluster.
+As an alternative to using the `configure-ovs.sh` shell script to set a `br-ex` bridge on a bare-metal platform, you can create a `MachineConfig` object that includes an NMState configuration file. The Kubernetes NMState Operator uses the NMState configuration file to create a customized `br-ex` bridge network configuration on each node in your cluster.
```

Comment:
Kubernetes-NMState isn't involved in the day 1 deployment flow. The machine-config is just applied by NMState directly on the host.

Reply:
:heavy_check_mark: